### DISCUSSION TITLE
Macintosh install problems and GitHub use problems

### DISCUSSION THREAD

**Discussion by fugueink** [dracula/dracula-theme#839] [Q&A] [answered]
I am sorry, but I have used GitHub for years (perhaps decades!) and never very well. I want to support Open Source software, but I am not a developer and cannot even figure out how to ask for help. Whenever I ask for help somewhere, I am told that isn't the place to do so, but no one suggests a venue that _is_ appropriate.

I cannot for the life of me get Dracula installed so that I can use it for LibreOffice for Mac. I am not even sure what directory I am supposed to be in when I use the git command (I suppose it generally goes without saying?). The git command seems to work, but then "Installing Palette and Theme" does not work.

Will someone please help me? I am using a 2013 MacBook Pro running MacOS 10.15.7 (19H2026) (latest version of Catalina).

Thank you for any help I can get here!

**Comment by benknoble** [comment, marked as answer]
I think the appropriate place is https://github.com/dracula/libreoffice/issues assuming that's the theme to which you refer.

In the meantime… unless there's a need to clone the files in a particular spot, which would have been indicated, you can run `git clone` anywhere. The cloned repository will then be in a sub-directory of the current directory (gently: https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository; manual page `git help clone` or https://git-scm.com/docs/git-clone).

As for the rest ("Installing Palette and Theme"): when you discuss this with the dracula/libreoffice team, it will be helpful to provide precise descriptions of what you did and what happened, including what worked and what didn't. People can't guess what you mean by "[it] does not work," so it's much harder for them to give appropriate suggestions to make something work until they have the full picture. Pun intended; pictures can be useful. But if it's text, prefer copy/pasting text to an image unless the issue you are describing is specifically a visual element of the text.
